openml/runs/run.py

```diff
@@ -165,7 +165,7 @@ def _create_description_xml(self):
         return description_xml
 
     @staticmethod
-    def _parse_parameters(model, flow):
+    def _parse_parameters(model, server_flow):
```

Comment:
Looking at this again I'm actually surprised that this is not called `run_task`, but only when publishing. But maybe this should be its own issue/PR.